pkg/common/ingress/controller/launch.go

```diff
@@ -34,44 +34,65 @@ func NewIngressController(backend ingress.Controller) *GenericController {
 	var (
 		flags = pflag.NewFlagSet("", pflag.ExitOnError)
 
-		apiserverHost = flags.String("apiserver-host", "", "The address of the Kubernetes Apiserver "+
-			"to connect to in the format of protocol://address:port, e.g., "+
-			"http://localhost:8080. If not specified, the assumption is that the binary runs inside a "+
-			"Kubernetes cluster and local discovery is attempted.")
+		apiserverHost = flags.String("apiserver-host", "",
+			`The address of the Kubernetes Apiserver to connect to in the format of
```

Comment:
```suggestion
			`The address of the Kubernetes API server to connect to, in the format of
```

I'm a bit pedantic about wording -- feel free to completely ignore me but i thought i'd clean up some of the wording a little, if you're modifying it already!

Reply:
Be my guest =) Because of that I open pull requests and wait a couple of days before merge - this way people has the chance to ping and ask "what?". I'll have a look into the comments shortly.

Reply:
Applied all the changes, thanks for the suggestions.